services/movie.py

```diff
@@ -0,0 +1,37 @@
+import init_django_orm  # noqa: F401
+
+from django.db.models.query import QuerySet
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
```

Comment:
If argument can be `None`, you should show it in type hint
```suggestion
def get_movies(genres_ids: list[int] | None = None,
               actors_ids: list[int] | None = None) -> QuerySet:
```